app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  scope :order_desc, ->{order created_at: :desc}
+  
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
và nói cho c sữ khác biệt nhé

Reply:
Dùng validate user vì:
Nếu validate: user_id, thì sẽ không chắc chắn được người dùng có user_id đấy có tồn tại thật hay không
Validate: user, sẽ chắc chắn được user tồn tại